--- wengophone-2.1.2-source/libs/owpixertool/include/pixertool/ffmpeg-pixertool.h.orig	2007-11-07 19:09:55.000000000 +0100
+++ wengophone-2.1.2-source/libs/owpixertool/include/pixertool/ffmpeg-pixertool.h	2007-11-07 19:10:18.000000000 +0100
@@ -22,12 +22,12 @@
 
 #include <pixertool/pixertool.h>
 
-#include <avcodec.h>
+
 
 #ifdef __cplusplus
 extern "C" {
 #endif
-
+#include <avcodec.h>
 /**
  * Palette identifier conversion.
  *
--- wengophone-2.1.2-source/wengophone/src/WengoPhoneBuildId.cpp.orig	2007-11-07 19:32:49.000000000 +0100
+++ wengophone-2.1.2-source/wengophone/src/WengoPhoneBuildId.cpp	2007-11-07 19:33:06.000000000 +0100
@@ -27,10 +27,10 @@
 	#include <portaudio.h>
 #endif
 #include <boost/version.hpp>
-#include <avcodec.h>
 #include <tinyxml.h>
 extern "C" {
 #include <glib.h>
+#include <avcodec.h>
 #include <gaim/core.h>
 }
 
